_includes/errors/404.php

```diff
@@ -3,10 +3,11 @@
 
   head([
     'title' => "Page not found",
-    'index' => false   
+    'index' => false
   ]);
   
-  $page = $_SERVER["REQUEST_URI"];
+  // IIS uses REQUEST_URI; Apache using uri
```

Comment:
```suggestion
  // IIS uses REQUEST_URI and _SERVER; Apache using uri and _REQUEST
```

I assume that's the way it is...